cuesubmit/tests/Submission_tests.py

```diff
@@ -49,6 +49,25 @@
     'services': ['nuke'],
 }
 
+BLENDER_MULTI_LAYER_DATA = {
+    'name': 'arbitraryBlenderLayer_name',
+    'layerType': cuesubmit.JobTypes.JobTypes.BLENDER,
+    'cmd': {'outputPath': '/path/to/output',
+            'blenderFile': '/path/to/scene.blend',
+            'outputFormat': 'PNG'},
+    'layerRange': '3-9',
+    'cores': '2'
```

Comment:
Do you need to add a `services` line like the Maya / Nuke tests do?

I think this is causing the job to default to the `shell` service, so the assertion on line 153 fails.

Reply:
@bcipriano noted, missed that one. 😄 
Resolved in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1337/commits/a17be62c8bd2c37067090db30bc495c66d59e32f

A new set of unit test failures seem to be coming from some unrelated changes.
Unsure, but I think the CueGUI fail is caused by this one: https://github.com/AcademySoftwareFoundation/OpenCue/commit/65b74fee2702a57a8dd10fb563c007d3963d585e
FYI @DiegoTavares

Reply:
Sorry about that, I'm fixing the pipeline on muster

Reply:
Pipeline fixed by https://github.com/AcademySoftwareFoundation/OpenCue/pull/1360

Reply:
Noted @DiegoTavares, thanks for the update!